xml/System.Net/IPAddress.xml

```diff
@@ -2044,10 +2074,10 @@
         <Parameter Name="bytesWritten" Type="System.Int32" RefType="out" Index="1" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="destination">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="destination">When this method returns, the IP address as a span of bytes.</param>
+        <param name="bytesWritten">When this method returns, the number of bytes written into the span.</param>
```

Comment:
```suggestion
        <param name="bytesWritten">When this method returns <see langword="true" />, receives the number of bytes written into the span.</param>
```

Reply:
I don't think this suggestion is correct. `bytesWritten` will always have a number, whether the method returns `true` or `false`. Usually these methods set this argument to `0` right before they return `false`. If that's the case, there's no point in mentioning "When this method returns true|false".

What @Jozkee wrote might be enough, if he can confirm that the method sets this to zero on failure.

Reply:
I prefer the original version better. So if it's simply 0 when false, the original sentence still looks good.